packages/base/src/index.js

```diff
@@ -163,6 +163,7 @@ module.exports = {
         vars: 'all',
         args: 'all',
         argsIgnorePattern: '[_]+',
```

Comment:
Kind of off topic but I think it's better to do this in the way they suggest in the rule docs:
```suggestion
        argsIgnorePattern: '^_'
```